spec/localeapp/cli/install_spec.rb

```diff
@@ -14,7 +14,7 @@
 
   it "executes the installer with the given key" do
     installer = double(:installer)
-    expect(installer).to receive(:execute).with(key)
+    expect(installer).to receive(:execute).with(key, false)
```

Comment:
I disagree, this test is about the "given key": `executes the installer with the given key`.
If we want/need to test the second argument, either we add a new example testing it, or we modify the appropriate test.